cartography/intel/gsuite/api.py

```diff
@@ -130,6 +130,14 @@ def load_gsuite_groups(neo4j_session: neo4j.Session, groups: List[Dict], gsuite_
         UNWIND $GroupData as group
         MERGE (g:GSuiteGroup{id: group.id})
         ON CREATE SET
+        g.group_id = group.id,
```

Comment:
Instead of having these as `ON CREATE SET` I think it makes sense to have them in `SET`.

`ON CREATE SET` is mostly for things like ID fields and firstseen.

Reply:
Indeed.